members/ashish/data.json

```diff
@@ -0,0 +1,10 @@
+{
+	"id": "sumitdhanania",
```

Comment:
ID is wrong

Reply:
@ankushdharkar  my id is right. https://github.com/ashishmangla0  is mu github url

Reply:
@ashishmangla0 But you have added the id "sumitdhanania". Please check.

Reply:
@ankushdharkar  my bad. make changes.